lib/Ix/Util.pm

```diff
@@ -19,6 +20,20 @@ use Sub::Exporter -setup => {
   ],
 };
 
+=head1 OVERVIEW
+
+This module exports (via L<Sub::Exporter>) a bunch of utility methods. More
+interestingly, It also includes C<Ix::DateTime>.
```

Comment:
It -> it unless Pennywise is the one including Ix::DateTime

Reply:
🤡